### ISSUE TITLE
Import if not in database

### ISSUE THREAD

**Issue by 3Descape** [Adldap2/Adldap2-Laravel#587]
//Not an issue but rather a help request//

Hello.
I was wondering if your package allows for importing the user, if he tries to sign in but the record is not synced into the database yet. Sort of "autoimporting".

The situation is the following: I have a set of about 800 users from our school. Of course it is possible that the school gets a new student, who should be able to use the website as soon as the recives a windows account, whithout the need for the admin to sync the database first.

Is this possible?

Thx in advance for helping me out and a super big thank you for creating and maintaining the package! It's super great!

**Comment by stevebauman** [comment]
Hi @3Descape, this is actually one of the main features of Adldap2-Laravel.

With the default configuration, a user record will be automatically created upon sign in.

You can prevent this from un-commenting the `OnlyImported` rule inside the stock configuration:

https://github.com/Adldap2/Adldap2-Laravel/blob/65f55f426d9407c794fe7b107ba902e586b00fd2/src/Config/auth.php#L55-L63

**Comment by stevebauman** [comment]
Also thanks for your kind words! :smile:

If you encounter issues just give me a shout and I'll help you out.

**Comment by 3Descape** [comment]
Thank you very much for the quick anwer!

While I was explaining how I tested it, because it didn't work, I found the solution: I implemented a custom login method in the LoginController, because I need to do some other checks for user permissions ans such, and there I validate whether or not the user exists. So it was my fault ;D 

Thank you very much again and keep up the great work! I really appreciate it!
